packages/core/src/content/content-client.ts

```diff
@@ -129,4 +135,83 @@ export class ContentClient {
     const response = await this.get<LocalesQueryResponse>(GET_LOCALES_QUERY);
     return response.manyLocale;
   }
+
+  /**
+   * Retrieves all available taxonomies with optional pagination support.
+   * @param {object} [options] - Optional pagination options.
+   * @param {number} [options.pageSize] - Limits the number of taxonomies returned per page. Defaults to the API's default
+   * @param {string} [options.after] - Cursor for pagination; use the `cursor` returned from the previous call to fetch the next page.
+   * @returns A promise that resolves to an object containing taxonomies, their terms, and pagination info.
+   */
+  async getTaxonomies(options?: { pageSize?: number; after?: string }) {
+    debug.content(
+      'Getting taxonomies (pageSize: %s, after: %s)',
+      options?.pageSize ?? 'API Default',
+      options?.after ?? ''
+    );
+
+    const variables: Record<string, any> = {
```

Comment:
`any` is not allowed to be used, you can just remove explicit type
```suggestion
    const variables = {
```